index.js

```diff
@@ -0,0 +1,200 @@
+              </tr>
+            </tbody>
+          </table>
+        </article>
+
+        <h4 id="trip-descriptor">Description</h4>
+        <article class="trip-details-about column small-8 small-centered row">
+          <p class="trip-about-text">${response.about}</p>
+        </article>
+
+        <div class="column small-8 small-centered row">
+          <button id="reserve-trip" class="button expanded"><strong>MAKE A RESERVATION</strong></button>
+
+          <section id="reservation-modal">
+            <form id="reservation" data-id="${response.id}">
+
+              <label for="name">Name:</label>
+              <input type="text" name="name" placeholder="Firstname Lastname" required="required"></input>
+
+              <label for="age">Age:</label>
+              <input type="number" name="age" required="required"></input>
+
+              <label for="email">Email:</label>
+              <input type="text" name="email" required="required"></input>
+
+              <button class="button expanded" type="submit"><strong>RESERVE TRIP</strong></button>
+            </form>
+          </section>
+        </div>
+      </section>
+      `;
+
+      $('#single-trip-info').html(singleTripTable);
+      $('#reservation-modal').hide();
+
+    }).fail(() => {
+      console.log('Did not load successfully!');
+      $('#featured-content').html('<p><em>An error has occurred. The trip details could not be loaded.</em></p>');
+    });
+  };
+
+  // FUNCTION FOR AJAX REQUEST AND RESPONSE TO RESERVE TRIP
+  const makeTripReservation = function makeTripReservation(id) {
+
+    const tripID = id;
+    const tripReservationsURL = `https://trektravel.herokuapp.com/trips/${tripID}/reservations`
+    console.log(tripReservationsURL);
+
+    const successCallback = function(response) {
+      console.log('POST request was successful');
+      console.log(response);
+      $('#reservation-modal').html('Your trips has successfully been reserved!');
+    };
+
+    const failCallback = function(response) {
+      console.log('An error occurred: Post was unsuccessful');
+      console.log(response);
+      $('#reservation-modal').html('Error: Your trip could not be reserved. Please refresh your browser and try again.');
+    };
+
+    let formData = $('#reservation').serialize();
+    console.log(formData);
+
+    $.post(tripReservationsURL, formData, successCallback).fail(failCallback);
+  };
+
+  // FOUNDATION
+  $(document).foundation();
+
+  // EVENTS
+  // To view All Trips
+  $('#all-trips').on('click', function() {
+    loadTrips();
+  });
+
+  // To View Trip Details
+  $('#featured-content').on('click', '#trip-details', function() {
+    //console.log('Button trip details clicked');
+    let tripID = $(this).attr('data-id');
+    getTripDetails(tripID);
+  });
+
+  // To Reserve Trip
+  $('#single-trip-info').on('submit', '#reservation', function(response) {
+    console.log('Button reserve trip clicked');
+    event.preventDefault();
```

Comment:
You're right-- the callback that you pass in as the third parameter to `.on(...)` should be a function that takes in one parameter. In this case, you name it `response`, but it _does_ represent the event object that describes the event that triggered this handler.

On this line, (line 169) you call `event.preventDefault();`. However, `event` is not defined within this scope, so the JS breaks at this point.

If you rename `response` to `event`, the app works as expected.